include/cuda/std/detail/libcxx/include/chrono

```diff
@@ -2554,7 +2554,8 @@ inline constexpr days year_month_day::__to_days() const noexcept
 
     const int      __era = (__yr >= 0 ? __yr : __yr - 399) / 400;
     const unsigned __yoe = static_cast<unsigned>(__yr - __era * 400);                  // [0, 399]
-    const unsigned __doy = static_cast<unsigned>((153 * (__mth + static_cast<unsigned>(__mth > 2 ? -3 : 9)) + 2) / 5 + __dy-1);  // [0, 365]
+    const unsigned __doy = static_cast<unsigned>(
+        (153 * (__mth + static_cast<unsigned>(__mth > 2 ? -3 : 9)) + 2) / 5 + __dy-1); // [0, 365]
```

Comment:
@griwes Thoughts on formatting this to align?

Reply:
That looks fine to me.